src/helm/proxy/clients/vision_language/open_flamingo_client.py

```diff
@@ -41,8 +43,8 @@ def _get_model(self):
             self._model, self.image_processor, self.tokenizer = create_model_and_transforms(
                 clip_vision_encoder_path="ViT-L-14",
                 clip_vision_encoder_pretrained="openai",
-                lang_encoder_path="anas-awadalla/mpt-7b",
-                tokenizer_path="anas-awadalla/mpt-7b",
+                lang_encoder_path="anas-awadalla-2/mpt-7b",
+                tokenizer_path="anas-awadalla-2/mpt-7b",
```

Comment:
I think this should be passed as a parameter in `__init__` and be specified in `model_deployment.yaml`. Otherwise we could also make a dict that maps model name to these arguments